frontend/awx/views/schedules/components/ScheduleResourceInputs.tsx

```diff
@@ -37,37 +38,65 @@ export function ScheduleResourceInputs() {
   }, [timeZone, t, links]);
 
   return (
-    <>
-      <PageFormSection>
+    <PageFormSection>
+      <PageFormTextInput<ScheduleFormWizard> name={'name'} isRequired label={t('Schedule name')} />
+      <PageFormTextInput<ScheduleFormWizard> name={'description'} label={t('Description')} />
+      <PageFormDateTimePicker<ScheduleFormWizard>
+        label={t('Start date/time')}
+        name="startDateTime"
+        isRequired
+      />
+      <PageFormSingleSelect<ScheduleFormWizard>
+        name="timezone"
+        placeholder={t('Select a time zone')}
+        label={t('Time zone')}
+        options={timeZones}
+        helperText={timezoneMessage}
+        isRequired
+      />
+      <PageFormSingleSelect
+        disableSortOptions
+        name="endType"
+        label={t('Schedule ending type')}
+        placeholder={t('Method used to stop schedule')}
+        options={[
+          { value: 'never', label: t('Never'), description: t('Never ending schedule') },
+          { value: 'count', label: t('Count'), description: t('Stop after a number of runs') },
+          { value: 'until', label: t('Until'), description: t('Stop on a specific date') },
+        ]}
+        isRequired
+      />
+      {endType === 'count' && (
         <PageFormTextInput<ScheduleFormWizard>
-          name={'name'}
+          labelHelpTitle={t('Count')}
+          label={t('Count')}
+          name="count"
+          placeholder="5"
+          labelHelp={t('The number of time this rule should be used.')}
+          min={0}
+          max={999}
+          type="number"
           isRequired
-          label={t('Schedule name')}
         />
-        <PageFormTextInput<ScheduleFormWizard> name={'description'} label={t('Description')} />
+      )}
+      {endType === 'until' && (
         <PageFormDateTimePicker<ScheduleFormWizard>
-          label={t('Start date/time')}
-          name="startDateTime"
-          isRequired
+          name="until"
+          timePlaceHolder="HH:MM AM/PM"
```

Comment:
Lets move this back to `RuleForm.tsx`.  I originally suggested we move it here trying to resolve that red herring of a bug.  Each rule can have an until or a count and